src/utils.ts

```diff
@@ -143,8 +138,8 @@ export const caseInsensitiveRegex = (input) => {
 export const inputXOR = (arg1, arg2) => {
   const [[key1, value1]] = Object.entries(arg1)
   const [[key2, value2]] = Object.entries(arg2)
-  if(!(!value1 != !value2)) {
-    throw new ValidationError(`Either ${key1} or ${key2} is required, but not both`, {logger: baseLogger});
+  if(!(!value1 !== !value2)) {
```

Comment:
@krtk6160 could there be a reason for the `==` here?

Reply:
Don't think there'll be an issue with this change. At the moment this is only used by the admin-panel (afaik) so there shouldn't be any impact on the users in case things go wrong.
Need to test this out, but there's an issue with the local dev setup scripts right now so unable to do so. Working on fixing that.

Reply:
The expression here is a triple-negative one! I'd love to flip it to just one negative.